### DISCUSSION TITLE
How to hide the flowise branding on embeded chatbot?

### DISCUSSION THREAD

**Discussion by remixdance** [FlowiseAI/Flowise#1351] [Q&A]
I want to hide the "Powerd by flowise "or rebrand my chatbot but with no localy I´m using render so i hope can guide me with render instructions thks¡

**Comment by kientv** [comment]
Remove it on .js file of chatbox

**Reply by remixdance** [reply]
sorry and where is located?

**Comment by kientv** [comment]
> sorry and where is located?

Search `Powered by` on that file and remove or change it.

**Reply by remixdance** [reply]
Yes I know, but where I can search the js file? I mean in my forked repository what is the name of the js file?

**Comment by blissiraDC** [comment]
If you want an easier fix, just make the powered-by font colour the same as your window background colour in the styling config.

**Reply by SmartMart69** [reply]
or just "transparent" ;-)

**Comment by chungyau97** [comment]
Hi @remixdance, have a look on our documentation 
https://docs.flowiseai.com/using-flowise/embed
